blocks/init/src/Blocks/components/modal/docs/readme.mdx

```diff
@@ -0,0 +1,11 @@
+# Modal Component
+
+Component used to provide modal functionality to your project.
+
+## Implementation
+
+Open a terminal and type in this command inside your projects root:
+
+```shell
+wp boilerplate use_component --name=modal
+```
```

Comment:
If you use the button component from frontend libs (as you should), you can specify it here as a dependency (see the card's readme for example / structure)